src/styles/_catalog.scss

```
@@ -1,22 +1,37 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$grid-gap: 48px;
+$small: 488px;
+$medium: 768px;
+$large: 1024px;
+$card-width: 200px;
 
+.catalog {
+  display: grid;
+  gap: $grid-gap $grid-gap;
+  grid-template-columns: repeat(4, $card-width);
+  justify-content: center;
   max-width: 944px;
   margin: 0 auto;
 
+  @media screen and (min-width: $medium) and (max-width: $large) {
```

```suggestion
  @media (max-width: $large) {
```
it would be enough. check all such cases